src/action/channel.js

```diff
@@ -168,7 +168,6 @@ class ChannelAction {
         status: 'waiting-close',
       }));
       this._store.pendingChannels = [].concat(pocs, pccs, pfccs, wccs);
-      this._store.limboBalanceSatoshis = response.totalLimboBalance;
```

Comment:
Ah so [this](https://github.com/lightninglabs/lightning-app/issues/1007#issuecomment-472722390) comment is outdated? Basically we were calculating correctly to begin with? I forget why we initially thought `limboBalance` was relevant to anything :thinking:

Reply:
Yes. They are one and the same. We can calculate limboBalanceSatoshis from the pendingChannels grpc response array.